docs/content/ingestion/tasks.md

```diff
@@ -245,10 +234,53 @@ The grammar is:
     "dataSource": <task_datasource>,
     "aggregations": <list of aggregators>,
     "rollup": <whether or not to rollup data during a merge>,
-    "interval": <DataSegment objects in this interval are going to be merged>
+    "interval": <DataSegment objects in this interval are going to be merged>,
+    "context": <task context>
 }
 ```
 
+### Compaction Task
+
+Compaction tasks merge all segments of the given interval. The syntax is:
```

Comment:
I think this should include a segmentGranularity too. Unless your idea is that the `interval` specified should just be one segment's worth of interval, in which case, that should be said in the docs.

Reply:
Yeah, all the segments of the ```interval``` are always merged into a single segment. I added the below statement.

> This compaction task merges _all segments_ of the interval `2017-01-01/2018-01-01` into a _single segment_.

Reply:
I suggest adding two more sentences:

> To merge each day's worth of data into a separate segment, you can submit multiple "compact" tasks, one for each day. They will run in parallel.

Reply:
Added.